e2e/tests/nightly/owner-copy-to-workspace.spec.ts

```diff
@@ -33,10 +33,7 @@ describe('Workspace owner can copy notebook', () => {
   const destinationDefaultCdrWorkspace = 'e2eNightlyOwnerCopyNotebookDestinationWorkspace';
   const destinationOldCdrWorkspace = 'e2eNightlyOwnerCopyNotebookDestinationOldCdrWorkspace';
 
-  /*Skipping the tests below as they will be moved to the new version of e2e test. Story tracking this
```

Comment:
still failing for me locally.  have not investigated yet.

Reply:
Checking

Reply:
Was there a Go to Copied Notebook link on COPY NOTEBOOK from one workspace to another? I cannot see it on modal but it seems the test is looking for it.

Reply:
copy-modal.tsx:
```
if (this.state.requestState === RequestState.SUCCESS) {
        const { namespace, id } = this.state.destination;
        return (
          <Button
            path={`/workspaces/${namespace}/${id}/${ResourceTypeHomeTabs.get(
              this.props.resourceType
            )}`}
            style={{ marginLeft: '0.5rem' }}
          >
            Go to Copied {resourceType}
          </Button>
        );
```

Reply:
We do have this code, but somehow it is does not work. in all envs, after copy, the copy modal closed without this option...

Reply:
I was able to test this manually. The test failed since it was pointing to test

Reply:
we are postponing this fix to https://precisionmedicineinitiative.atlassian.net/browse/RW-9034 as noted in the comments